### ISSUE TITLE
Tab reload crash

### ISSUE THREAD

**Issue by clragon** [GitSquared/edex-ui#630]
## Technical information
**Using version:**
 - `master` (running from GitHub-published source code, currently `v3.0.0-pre`)

**Running on:**
 - Windows

**How comfortable you are with your system and/or IT in general:**
 - My machine is fully under my control, tell me what you need

---

## Problem

Reloading the app and opening a previously open terminal tab leads to a crash.

![12774246](https://user-images.githubusercontent.com/11785085/68545685-a46de300-03cf-11ea-981f-bbd9a9a8f57f.png)

**Comment by GitSquared** [comment]
Tabs state isn't saved between window reloads, which causes this.
Perhaps the main process could forward a list of open sockets via IPC?
Will look into this

**Comment by Animus-Surge** [comment]
Might have already fixed this in #892

**Comment by GitSquared** [comment]
Indeed, it doesn't crash anymore... The state is still not saved between UI reloads though, so I think we can keep this open.

Unlikely to ever be fixed before a rewrite, to be honest.

**Comment by Animus-Surge** [comment]
> The state is still not saved between UI reloads

Can look into that if you want. Doesn’t seem like it should be too hard (famous last words)

**Comment by GitSquared** [comment]
haha, sure, feel free to try it.

**Comment by kooshlash** [comment]
Mailbox
